### ISSUE TITLE
Wrongly parsed docstrings

### ISSUE THREAD

**Issue by BartGo** [bndr/pipreqs#35]
Pipreqs 0.3.8, Windows 10, my file example/main.py imports "bottle".

```
"""
Evil docstring
"""

print "Not a comment"
import bottle

"""
print "A comment"
import peewee
"""
```

Running:

```
> python example/main.py
Not a comment

> pipreqs --debug example
DEBUG: Found packages: set([])
DEBUG: Found imports:
DEBUG: Getting packages information from Local/PyPI
DEBUG: Writing 0 requirements:  to example\requirements.txt
INFO: Successfully saved requirements file in example\requirements.txt
```

Removal of the starting comment fixes the problem (bottle found).

**Comment by bndr** [comment]
Thanks for reporting, this should be fixed in 0.3.9.

If you find any more bugs, please report.

**Comment by BartGo** [comment]
Thanks for the fix, it works but there seems to be a side effect or a related issue - I reported it as a separate ticket  #36.
